### PR TITLE
Rspec

### PR DESCRIPTION
**Pull request by misterpaul4**
- tested classes methods for win.rb and board.rb

### CODE REVIEW

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great Start @team :clap: :clap: However, some requirements need to be implemented in order to get approval for this project. Check :arrow_down: 

* [ ] All the public methods of the classes implemented in project OOP are tested ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/all_tic_tac_toe.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/))
  - Two methods in Board class weren't tested.
  - In win_spec.rb, you should use the Board (e.g. Board.new) class to test the Win Class.
* [ ] specs cover positive and negative scenarios ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/Ruby_testing/commonsense.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59566098/)) 
  -Example: You could have tested more winning and draw cases.
 
* [ ] Using `require_relative` for files in your directory instead of `require` is a ruby convention.

> "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."

- **[SUGGESTION]** Since you implemented testing, you could add a section in you readme explaining How to run the tests locally and what are you testing.

### That's it :relieved: When finished, kindly submit a code re-review request :pray: 
**_Ahmed Mahfoudh_**

**Review by mosesogwo — APPROVED** [review]
**Hi @misterpaul4, @Ferm0494, great job:clap:. I'll approve this but you must make these changes before you merge:**
:point_right: The previous reviewer asked you to add a testing section to your readme - it doesn't mean you should remove the game instructions. Please return the Getting Started section together with the current section for Testing.
:point_right: You still have require in board_spec where you could have used require_relative.
:point_right: You have a commented line in win_spec:

These are small changes. Please make them before merging.

### Status: Approved:heavy_check_mark:

**Pull request merged**
